src/components/TodoList/TodoList.tsx

```diff
@@ -1,44 +1,147 @@
 import React from 'react';
 import './TodoList.scss';
+import classnames from 'classnames';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
+type Props = {
+  todos: Todo[];
+  selectUser: (selectedUserId:number, todoId:number) => void;
+  todoId: number;
+};
 
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
+type State = {
+  title: string;
+  todoList: Todo[];
+  todosStatus: string;
+};
 
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
+export class TodoList extends React.Component<Props, State> {
+  state: State = {
+    title: '',
+    todoList: [],
+    todosStatus: '',
+  };
+
+  static getDerivedStateFromProps(nextProps: Props, prevState: State) {
+    if (prevState.todoList !== nextProps.todos) {
+      return {
+        todoList: nextProps.todos,
+      };
+    }
+
+    return null;
+  }
+
+  handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    this.setState({ title: e.target.value });
+  };
+
+  filterTodosByTitle = () => {
```

Comment:
This function returns not just filtered movies by title. It returns filtered movies at all